### PR TITLE
Lower fixes

### PR DESCRIPTION
**Pull request by Boniara**

### CODE REVIEW

File: sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java [hunk 1]
```diff
@@ -104,10 +105,18 @@ public void initDb() {
 
     @RabbitListener(queues = "#{settingsQueue.name}")
     public void process(Message message) {
-        ReinitEventMessage rm = new Gson().fromJson(new String(message.getBody()), ReinitEventMessage.class);
-        if (!eventPushService.isSettingQueueConsumer(message)) {
-            Integration integration = integrationService.retrieveById(rm.getIntegrationId());
-            integrationInitializer.initIntegration(integration, rm.getTenancy());
+        try {
+            ReinitEventMessage rm = new Gson().fromJson(new String(message.getBody()), ReinitEventMessage.class);
```
Comment (qps-ua-nsidorevich) [inline sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java]: Rename rm to something more meaningful.

Comment (nsidorevich) [inline sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java]: Please rename rm to something more meaningful.

File: sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java [hunk 2]
```diff
@@ -104,10 +105,18 @@ public void initDb() {
...
+            if (!eventPushService.isSettingQueueConsumer(message)) {
+                TenancyContext.setTenantName(rm.getTenancy());
+
+                Integration integration = integrationService.retrieveById(rm.getIntegrationId());
+                integrationInitializer.initIntegration(integration, rm.getTenancy());
+
+                TenancyContext.setTenantName(null);
+            }
+        } catch (Exception e) {
+            LOGGER.error("Unable to initialize adapter. " + e.getMessage(), e);
```
Comment (nsidorevich) [inline sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java]: What exactly might cause an exception here? If message is not malformed (can we add a separate catch?) can we obtain from message and log something useful for us like integration id at least?

File: sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java [hunk 3]
```diff
@@ -104,10 +105,18 @@ public void initDb() {
...
-            integrationInitializer.initIntegration(integration, rm.getTenancy());
+        try {
+            ReinitEventMessage rm = new Gson().fromJson(new String(message.getBody()), ReinitEventMessage.class);
+            if (!eventPushService.isSettingQueueConsumer(message)) {
+                TenancyContext.setTenantName(rm.getTenancy());
+
+                Integration integration = integrationService.retrieveById(rm.getIntegrationId());
+                integrationInitializer.initIntegration(integration, rm.getTenancy());
+
+                TenancyContext.setTenantName(null);
```
Comment (tsvrko) [inline sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java]: What the problem we usually fix setting tenant name as null? Noticed it in a coupe other places, once even as causing bug.

Comment (Boniara) [reply sources/zafira-services/src/main/java/com/qaprosoft/zafira/service/integration/core/IntegrationTenancyStorage.java]: @tsvrko no problems, just be sure that thread maybe stored in a rabbit pool does not have a tenant state (additional check)

**Pull request merged**